Content.Shared/DeviceLinking/DeviceLinkSourceComponent.cs

```diff
@@ -20,6 +20,14 @@ public sealed partial class DeviceLinkSourceComponent : Component
     [ViewVariables]
     public Dictionary<string, HashSet<EntityUid>> Outputs = new();
 
+    /// <summary>
+    /// If set to High or Low, the last signal state for a given port.
+    /// Used when linking ports of devices that are currently outputting a signal.
+    /// Only set by <c>DeviceLinkSystem.SendSignal</c>.
+    /// </summary>
+    [DataField]
+    public Dictionary<string, bool> LastSignals = new();
+
```

Comment:
Is this supposed to be NetworkPayload or a dictionary.

Reply:
a dictionary idk if ProtoId should be used

Reply:
using protoid now